store/src/java-test/com/zimbra/cs/mailbox/MailboxTestUtil.java

```diff
@@ -85,33 +85,35 @@ public static void initProvisioning() throws Exception {
      * @throws Exception
      */
     public static void initProvisioning(String zimbraServerDir) throws Exception {
-        String zimbraHome = "/opt/zimbra/";
-        zimbraServerDir = Strings.nullToEmpty(zimbraServerDir);
-        System.setProperty("log4j.configuration", "log4j-test.properties");
-        // Don't load from /opt/zimbra/conf
-        String pathPrefix = "/";
-        // If zimbraServerDir is empty, use relative path to localconfig-test.xml
-        if (StringUtils.isEmpty(zimbraServerDir)) {
-            pathPrefix = "";
-        }
-        if (!StringUtil.isEmpty(zimbraServerDir) && !zimbraServerDir.endsWith("/")) {
-            zimbraServerDir = zimbraServerDir + "/";
-        }
-        System.setProperty("zimbra.config", zimbraServerDir + pathPrefix + "src/java-test/localconfig-test.xml");
-        LC.reload();
-        if (Strings.isNullOrEmpty(zimbraServerDir)) {
-            zimbraServerDir = System.getProperty("user.dir");
+    zimbraServerDir = getZimbraServerDir(zimbraServerDir);
+    System.setProperty("log4j.configuration", "log4j-test.properties");
+    System.setProperty("zimbra.config", zimbraServerDir + "src/java-test/localconfig-test.xml");
+    LC.reload();
+
+    String timezonefilePath = zimbraServerDir + "src/java-test/timezones.ics";
```

Comment:
Can we call this timezones-test.ics, that we are very clear this is just for unit test purpose.